cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -0,0 +1,244 @@
+{% import 'molecules/hero.html' as hero %}
+{% import 'molecules/image-text-25-75.html' as image_text_25_75 %}
+{% import 'molecules/image-text-50-50.html' as image_text_50_50 %}
+{% import 'molecules/related-posts.html' as related_posts %}
+{% import 'molecules/text-introduction.html' as text_introduction %}
+
+{% import 'organisms/email-signup.html' as email_signup %}
+{% import 'organisms/full-width-text.html' as full_width_text %}
+{% import 'organisms/main-contact-info.html' as main_contact_info %}
+{% import 'organisms/post-preview.html' as post_preview %}
+{% import 'organisms/sidebar-breakout.html' as sidebar_breakout %}
+{% import 'organisms/sidebar-contact-info.html' as sidebar_contact_info %}
+{% import 'organisms/table.html' as table %}
+{% import 'organisms/well.html' as well %}
+
+{% block title -%}
+    TODO: This is a prototype page for testing landing page layouts.
+    Remove when fully implemented elsewhere.
+{%- endblock %}
+
+{# NOTE: There are three options for introductions:
+    - Only Intro (no Text intro or breakout header)
+    - Only Text Introduction (No Hero)
+    - Text Introduction and Breakout header #}
+
+{# TODO: Remove lipsumText when and if it is no longer needed. #}
+{% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+
+{% block hero -%}
+    {{ hero.render({
+        'heading':   'This is an example of a Hero Molecule',
+        'body':      'Heroes function as a primary focal point on a page,
+                      often used to introduce a collection of pages by
+                      combining a brief description of the goals of that
+                      section along with a visually impactful graphic.',
+        'image_url': 'http://www.consumerfinance.gov/know-before-you-owe/static/img/kbyo-hero_tablet-2x.png'
+    }, {
+        'background_color': '#d5e7e6',
+        'link_text':        'Hero call to action',
+        'link_url':         '#'
+    }) }}
+{% endblock %}
+
+{% block pre_content %}
+   {% set settings = {
+        'heading': 'Sample Header(H3)',
+        'image': {
+            'url':      '/static/img/clipboard-round-300x300.png',
+            'is_round': true
+        },
+        'intro':   {
+            'heading': 'Sample Header(H4)',
+            'body':    lipsumText,
+            'icon':    'speech-bubble'
+        },
+        'related_posts': {
+            'view_more': {
+                'link_url':  '/',
+                'link_text': 'View More Items'
+            },
+            'slug': 'four-years-working-for-you'
+        }
+       }
+    %}
+    {% call sidebar_breakout.render(settings) %}
+        {{ text_introduction.render({
+            'heading':   'Text Introduction',
+            'intro':     'Content Intro Lorem Ipsum',
+            'body':      lipsumText,
+            'link_text': 'Placeholder link',
+            'link_url':  '#'
+        }) }}
+    {% endcall %}
+{% endblock %}
+
+{% block content_main %}
+    <div class="o-image-text-50-50-group">
```

Comment:
Oh I thought I'd added it way back and then realized there isn't an organism for it.